### ISSUE TITLE
Static files can use a database?

### ISSUE THREAD

**Issue by alexislefebvre** [CleverCloud/documentation#11]
The “Deploy static files” page explains that we can [use environment variables](https://www.clever-cloud.com/doc/deploy/application/static/static/#setting-up-environment-variables-on-clever-cloud) and a [database](https://www.clever-cloud.com/doc/deploy/application/static/static/#linking-a-database-or-any-other-add-on-to-your-application).

I wonder how is it possible to use a database with static content? Does it mean that we can use a script in PHP, node, etc. to build the static content? This is not explained, but if it's possible, could you please explain how?

**Comment by juliamrch** [comment]
Yes, there are ways to incorporate dynamic data into a static site and to use a Database. Astro and T3 apps are a good example of this. This issue seems out of scope for our documentation at the moment, but I recommend reading about [Incremental Static Regeneration](https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration) to understand more about this.
